code/modules/modular_computers/file_system/programs/borgUI.dm

```diff
@@ -0,0 +1,140 @@
+/datum/computer_file/program/borgUI //generic parent that handles most of the process
```

Comment:
```suggestion
/datum/computer_file/program/borg_ui //generic parent that handles most of the process
```
snake case, please
this will require updating other parts of the code

Reply:
App name has been changed from borgUI to RoboTact. Path is `../program/robotact` now.